test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -668,7 +668,8 @@ Function sys.uniqueidentifier_cmp(sys.uniqueidentifier,sys.uniqueidentifier)
 Function sys.uniqueidentifier_hash(sys.uniqueidentifier)
 Function sys.uniqueidentifier_sqlvariant(sys.uniqueidentifier)
 Function sys.update(text)
-Function sys.user_id(text)
+Function sys.user_id()
```

Comment:
Please add upgrade tests to verify views/functions/procs dependency on this function work as expected. This is important as we doing a rename here.